src/blog/2023/07/flowforge-1-9-release.md

```diff
@@ -0,0 +1,70 @@
+---
+title: FlowForge now offers API Documentation with Swagger UI
+subtitle: FlowForge 1.9 adds new features to make it easier to administer FlowForge
+description: FlowForge 1.9 
+date: 2023-07-06 
+authors: ["ian-skerrett"]
+image: 
+tags:
+    - posts
+    - flowforge
+    - releases
+---
+
+FlowForge 1.9 adds new features to make it easier to adminster FlowForge platform deployments, including new API documentation and the ability to create customized Node-RED palettes.
+
+<!--more-->
+
+## API Documentation
```

Comment:
Once we've published the release, the docs at https://flowforge.com/docs/api/ will include more information including links to the API spec hosted of FF Cloud.

Reply:
To make sure I understand, the API spec for FF Cloud will allow people to create scripts against their FF Cloud accounts?